.travis.yml

```diff
@@ -1,76 +1,91 @@
-# We deliberately don't use travis's language=python option because
-# we install miniconda and use conda to get python. Additionally, 
-# Travis's auto-install of python doesn't work on osx images (see
-# https://github.com/travis-ci/travis-ci/issues/4729).
+# TODO before merge:
+#
+#  * try switch to defaults
+#
+#  * restore previous sections
 language: generic
```

Comment:
I'm going to go ahead and merge this, but I'll change "TODO before merge" to just "TODO".